content/en/about/version-policy/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Versioning Policy
+description: Supported versions of various components and dependencies of Istio.
+weight: 15
+icon: cadence
+---
+
+## Istio Versioning
+
+Istio versions are expressed as **x**.**y**.**z**, where **x** is the major version, **y** is the minor version, and **z** is the patch version.
+
+Istio provides support for each minor release for 3 months after the next minor release. For example, Istio 1.0 support will be dropped 3 months after Istio 1.2 is released. See [Build & Release Cadence](/about/release-cadence/) for more details.
+
+## Component Version Skew
+
+Various components of Istio that interact with each other must be kept within supported version ranges. Below describes the required version skews. Where possible, we always recommend using the same version between all components.
+
+### Istioctl
+
+We currently recommend using `istioctl` within the same minor version as both the control plane and data plane.
+
+For example, `istioctl` version **1.0.1** could be used with control plane version **1.0.0** and data plane version **1.0.6**.
+
+### Control/Data Plane
+
+The data plane (both sidecars and gateways) must be either the same version as the control plane, or at most 1 minor version older.
+
+For example, the data plane version **1.0.0** could be used with control plane version **1.0.0** or **1.1.0**.
+
+This implies that the control plane must be upgraded first. For more information, see the [Upgrade](/docs/setup/upgrade/) documentation.
+
```

Comment:
does this imply people can't upgrade from 1.4 to 1.6?  Also, does this imply multiple control planes should be used when skipping 1 minor version upgrade.

Reply:
it would be good to provide some guidance msg around this.

Reply:
Yes! I believe this has always been out policy, and it is currently documented as well: https://istio.io/docs/setup/upgrade/cni-helm-upgrade/ "Istio does NOT support skip level upgrades. Only upgrades from 1.4 to 1.5 are supported. If you are on an older version, please upgrade to 1.4 first."

This is explicitly not mentioning use of multiple control planes to skip minor versions, because (1) this PR is meant to capture our current policy, which is no skip level updates (2) I am not (yet) convinced its a good idea

Reply:
Ok so multiple cps are recommended with 1 minor version upgrade.  They are not recommended for skip level update till further testing proves it.